src/main/java/seedu/lifeasier/commands/DeleteNotesCommand.java

```diff
@@ -49,16 +50,16 @@ private void findTitle(Ui ui, NoteList notes, String title, NoteHistory noteHist
             ui.showMultipleMatchesFoundMessage();
 
             logger.log(Level.INFO, "Start of printing all matching notes");
-            ui.printMultipleNoteMatches(notes, title);
+            matchArr = ui.printMultipleNoteMatches(notes, title, matchArr);
             logger.log(Level.INFO, "End of printing all matching notes");
 
             noteNumber = Integer.parseInt(ui.readCommand());
```

Comment:
Would you consider adding the "-1" logic in this line instead of in all the other times you use noteNumber?
```suggestion
            noteNumber = Integer.parseInt(ui.readCommand()) - 1;
```